deploy/stacks/pipeline.py

```diff
@@ -463,6 +463,38 @@ def set_quality_gate_stage(self):
                     security_groups=[self.codebuild_sg],
                 ),
             )
+        else:
+            it_project_role = iam.Role(
+                self,
+                id=f'ItCobdeBuildRole{self.git_branch}',
+                role_name=f'{self.resource_prefix}-{self.git_branch}-integration-tests-role',
+                assumed_by=iam.CompositePrincipal(
+                    iam.ServicePrincipal('codebuild.amazonaws.com'),
+                    iam.AccountPrincipal(self.account),
+                ),
+            )
+            for policy in self.codebuild_policy:
+                it_project_role.add_to_policy(policy)
+
+            gate_quality_wave = self.pipeline.add_wave('QualityGate')
```

Comment:
Can we change the id to `UploadToS3` instead of `QualityGate` to avoid confusion when quality gate set to false - otherwise looks good

Reply:
You are right. Added :)